docs/en/enterprise-edition/policy-reference/supply-chain-policies/sca-policies/vul-2.adoc

```diff
@@ -17,4 +17,4 @@
 === Description 
 
 
-Prisma Cloud has detected a known vulnerability in an open source package declared in a package manager file.
\ No newline at end of file
+Prisma Cloud has detected a known vulnerability in an open source package declared in a package manager file or container image.
```

Comment:
Prisma Cloud has detected a known vulnerability (CVE) in an open source package declared in a package manager file or container image.

Reply:
Updated. Please review